packages/xo-server/config.toml

```diff
@@ -87,8 +88,19 @@ snapshotNameLabelTpl = '[XO Backup {job.name}] {vm.name_label}'
 # Delay for which backups listing on a remote is cached
 listingDebounce = '1 min'
 
+# settings when using Vhd directories ( s3 , encryption )
+# you should use 'none' if your fs is already compressed
+# changing this setting will generate new full backups
 vhdDirectoryCompression = 'brotli'
 
+# how many block can be merged in parallel per backup running
+# increase to increase performance, reduce if you have timeout during merge
+mergeBlockConcurrency = 2
+
+# how many block can be uploaded in parallel
+# increase to in rease performance, reduce if you have timeout or memory error during transfer
+writeBlockConcurrency = 16
+
```

Comment:
As these settings are for a libraries and are used in two different apps (`xo-server` & `xo-proxy`), I feel like we should provide defaults values and document them somewhere else.

I think I will merge your PR as it is and then move them directly in a centralized, would that be ok for you?

Reply:
Is "2" enough as default? I think we tried with 4 with success and it was "fast enough" vs 2 (when we tested on Cécile's S3). Or maybe it was 2, I don't remember exactly :thinking:

Reply:
we tested 4 , but the results came after the patch release.

Reply:
Should we change that to 4 then?

Reply:
I think we shuold let this as is in this PR that only expose existing values
It will be easier to have some user tests  by changing the configuration and optimize the default values later.

Reply:
@fbeauchamp the config file is not good, because it's in `xo-server` and not in the lib that really uses these entries. Also, this does not make the settings easily discoverable for `xo-proxy`.

I think a dedicated `@xen-orchestra/backups/config.js` file would be a good first answer.

Reply:
yes, good idea